src/triggers.js

```diff
@@ -98,6 +98,27 @@ function getStore(category, name, applicationId) {
 function add(category, name, handler, applicationId) {
   const lastComponent = name.split('.').splice(-1);
   const store = getStore(category, name, applicationId);
+  if (store[lastComponent]) {
+    const type = name.split('.')[0];
+    let warningMsg =
+      type == lastComponent ? `${lastComponent}` : `trigger ${type}`;
+    const classSpecific = [
+      'beforeSave',
+      'afterSave',
+      'beforeDelete',
+      'afterDelete',
+      'beforeFind',
+      'afterFind',
+      'beforeSubscribe',
+      'afterEvent',
+    ];
+    if (classSpecific.includes(type)) {
+      warningMsg += ` on class ${lastComponent}`;
```

Comment:
I believe the message will not be printed how you expect because here you don't have only the class name. You have the original `lastComponent` that was passed to the function. I'd actually not do all this logic and just log `beforeSave.User`.

Reply:
Keep it simple hey!